app/Providers/RouteServiceProvider.php

```diff
@@ -54,7 +54,7 @@ public function boot()
      */
     protected function configureRateLimiting()
     {
-        RateLimiter::for('api', function (Request $request) {
+        RateLimiter::for('api', function () {
```

Comment:
Does this not make it worse? The request isn't used in the function.

Reply:
I removed it since it isn't used in the function, not sure how it make it worse but maybe I'm missing something

Reply:
@WindOfRussia Does the function need to have the parameter?

Reply:
nah, the parameter is only there by default to remind users they can customize the rate limiter based on some information from the request. hypothetical example: user-tier free is limiter to 10 api requests a second but paid users of the said system can have more